storage/database/dynamodb_test.go

```diff
@@ -112,15 +112,16 @@ func (s *SuiteDynamoDB) TestDynamoDB_Timeout() {
 		}
 		defer listen.Close()
 
-		for {
+		for i := 0; i < maxRetries+1; i++ {
```

Comment:
What would be different if we set `maxRetries` to 0 and remove the loop?

Reply:
@ehnuje The test ensures client timeout duration is the same as `time.Duration(maxRetries+1) * singleTimeout`. If `maxRetries` is zero, this TC cannot ensures that the retries have sequentially occurred and each try waits for singleTimeout.